src/m_config.c

```diff
@@ -1063,7 +1063,7 @@ static default_t extra_defaults_list[] =
     // MIDI device for native Windows MIDI.
     //
 
-    CONFIG_VARIABLE_STRING(winmm_midi_device),
+    CONFIG_VARIABLE_STRING(winmm_device),
```

Comment:
In my opinion, it's fine to change this since no release of Chocolate Doom has used `winmm_midi_device` yet. In fact, the last release is still using the obsolete "midiproc" implementation. Is that just passing the problem to forks? Yes, but the default behavior is to use the default Windows midi device with no input needed from the user. The most popular Windows midi device is already the default one (MS GS Synth) since most songs are composed with that in mind. The second most popular Windows midi device, VirtualMidiSynth, is also typically set as the default since it can do so directly from its config window. That leaves power users with VSTs or real hardware (e.g. SC-55) who can surely handle running *setup.exe if they were able to get those devices working.

Reply:
Frankly, I'm a little puzzled by this change. `winmm_midi_device` and `winmm_device` aren't that much different, so why complicate things for the few Crispy users who use this feature?

Reply:
Okay, removed the name change.